internal/scorecard/alpha/bundle.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func getBundleData(bundlePath string) (bundleData []byte, err error) {
+
+	// make sure the bundle exists on disk
+	_, err = os.Stat(bundlePath)
+	if os.IsNotExist(err) {
+		return bundleData, fmt.Errorf("bundle path is not valid %s", err.Error())
+	}
+
+	paths := []string{bundlePath}
+	err = Tartar("/tmp/my.tar", paths)
+	if err != nil {
+		return bundleData, fmt.Errorf("error creating tar of bundle %s", err.Error())
+	}
+
+	var buf []byte
+	buf, err = ioutil.ReadFile("/tmp/my.tar")
```

Comment:
could `/tmp/my.tar` be an const /

Reply:
Also, could we give a more significant name of my.tar? I think it will be deleted ... but just in case of not and to also to make easier we understand what is the `my.tar`.

Reply:
good catch, I updated this to have a name of tempBundle-xxxx.tar with the xxxx being randomly generated to avoid a file name collision, I also forgot to remove the file so I added a os.Remove() in there to do that.